### PR TITLE
Add label to open with option

### PR DESCRIPTION
**Pull request by jburel**
Add label to open with option
Nothing to test
This is the set-up we usually use

### CODE REVIEW

**Comment by pwalczysko** [comment]
see https://trello.com/c/OOdPOerR/6-order-of-images-selected-in-web-is-illogical-in-iviewer

**Comment by pwalczysko** [comment]
see https://trello.com/c/DxUKov4y/7-intensity-projection-range-selection

**Comment by pwalczysko** [comment]
see https://trello.com/c/8LFAARBr/8-rfe-projection-saving

**Comment by pwalczysko** [comment]
see https://trello.com/c/ocMfVaxD/9-rfe-multiselection-of-rois

**Comment by pwalczysko** [comment]
see https://trello.com/c/iH0uoUYZ/10-copy-and-paste-between-image-of-same-sizes-still-works-randomly

**Comment by pwalczysko** [comment]
see https://trello.com/c/hm84brEX/11-copy-and-paste-on-the-same-image-works-randomly-if-rhp-menu-is-used

**Comment by pwalczysko** [comment]
For the short feature list:

``support for floating images with very small pixels range`` -> improved controls for rendering of floating images with very small pixels range

**Comment by pwalczysko** [comment]
``add shortcut to draw line`` -> ``drawing lines now possible``

**Comment by pwalczysko** [comment]
``improve copy and paste of ROIs`` -> ``improve copy and paste of ROIs and add a new context menu for these actions``

**Comment by pwalczysko** [comment]
``add support for new Lookup tables`` -> ``add support for 2 new Lookup tables``

**Comment by pwalczysko** [comment]
``ability to open several images`` -> ``ability to open multiple selected images``

**Comment by pwalczysko** [comment]
Done with the ivewer review.

**Comment by jburel** [comment]
@pwalczysko I have updated the text as suggested

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,14 +4,15 @@
...
-- add shortcut to draw line
-- improve copy and paste of ROIs
-- add support for new Lookup tables
-- ability to open several images
-- improve layout of rois table
+- added ability to draw line
+- improved copy and paste of ROIs and add a new context menu for these actions
+- added support for new Lookup tables
+- added ability to open multiple selected images
+- improved layout of rois table
```
Comment (pwalczysko) [inline CHANGELOG.md]: maybe rois -> ROIs  (unify with the capitalized version three lines above this one)

**Comment by pwalczysko** [comment]
One small comment. Otherwise ready to merge.

**Comment by jburel** [comment]
@pwalczysko adjusted as requested

**Comment by pwalczysko** [comment]
Ready to merge.

**Comment by jburel** [comment]
To be merged when we are ready to release

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
+0.2.0 (June 2017)
+----------------
+
+This version includes:
+
+- support for omero prefix
+- improved manipulation of rendering setting for floating images with very small pixels range
+- support for maximum intensity projection along Z
+- loaded ROIS using the web api
```
Comment (hflynn) [inline CHANGELOG.md]: loaded ROIS -> loading ROIs

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,19 @@
...
+- added ability to draw line
```
Comment (hflynn) [inline CHANGELOG.md]: line -> lines

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,19 @@
...
+- improved copy and paste of ROIs and add a new context menu for these actions
```
Comment (hflynn) [inline CHANGELOG.md]: add -> added

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,9 +6,9 @@ This version includes:
 - support for omero prefix
 - improved manipulation of rendering setting for floating images with very small pixels range
 - support for maximum intensity projection along Z
-- loaded ROIS using the web api
-- added ability to draw line
-- improved copy and paste of ROIs and add a new context menu for these actions
+- loading ROIS using the web api
```
Comment (hflynn) [inline CHANGELOG.md]: You missed the capital S here - should be lowercase

File: README.rst [hunk 6]
```diff
@@ -6,7 +6,7 @@ An OMERO.web app for visualizing images in OMERO.
 Requirements
 ============
 
-* OMERO 5.3.0 or newer.
+* OMERO 5.3.2 or newer.
```
Comment (hflynn) [inline README.rst]: 5.3.3

**Pull request merged**